modules/sdk-coin-sol/src/lib/transferBuilderV2.ts

```diff
@@ -98,7 +131,7 @@ export class TransferBuilderV2 extends TransactionBuilder {
               fromAddress: this._sender,
               toAddress: sendParams.address,
               amount: sendParams.amount,
-              tokenName: sendParams.tokenName,
+              tokenName: coin.name,
```

Comment:
nit: why did this change ? seems like both are the same

Reply:
yes, both are same, in this block (can see the below return) I was using the `tokenName: coin.name`, so referential similar changed it.